api/image-parser.php

```diff
@@ -199,11 +207,19 @@ function get_extracted_images() {
  * @param  string  $url URL to be passed which is to be checked
  * @return boolean      returns if URL passed is valid or not
  */
-function isValidURL($url){
-    return preg_match('|^http(s)?://[a-z0-9-]+(.[a-z0-9-]+)*(:[0-9]+)?(/.*)?$|i', $url);
+// function isValidURL($url){
+//     return preg_match('|^http(s)?://[a-z0-9-]+(.[a-z0-9-]+)*(:[0-9]+)?(/.*)?$|i', $url);
+// }
+
+function validate_url($url) {
```

Comment:
I don't think there was any need of changing the function name
Please change it back to existing name